redis/include/userver/storages/redis/impl/base.hpp

```diff
@@ -27,16 +27,19 @@ struct ConnectionInfo {
   bool read_only = false;
   ConnectionSecurity connection_security = ConnectionSecurity::kNone;
   using HostVector = std::vector<std::string>;
+  std::optional<size_t> database_index{};
```

Comment:
New connections always use database index 0 so it is not really an optional:
```suggestion
  std::size_t database_index = 0;
```

Reply:
Done